Contibution File (#5)
kokoaespol/ChocoAPI: CONTRIBUTING.md

# Contributing file

> Thanks you for contribute to this "little" project for our club at ESPOL. If you are
> interested on suggest new functionalities you can do this with an issue, all new 
> things are welcome!!.

Reviewer: ```suggestion
Thank you for contributing to this "little" project for our club at ESPOL. If you are
interested on suggesting new features, you can do so with an issue. We welcome all suggestions!
```

### New Contributor guide

For an overview about this project you can read the [README](README.md) file.

This project is enterely developed using all the features that the language, Rust,
provide for us, and there are some resources that would help to you how Rust works
or how to do certains things:

TODO

### Project Code Style Guides

This project contains configuration files that would help to you to develop in the way
that we do. We recommend the use of the next plugins (if there's available for your IDE/Editor):

TODO

### Pull request

All the pull request are welcome, but ensure that they are concise and show what is the
new feature, bug fix, etc. to be included in our project, this would help to our team
to easily report changes or approve your pull request. 

Every pull request must have the next template:
```
Title: A new great feature!!!

Description:
This is an incredible feature, this will help ....
```

> The **Title** need to show a resume or a name for the new change/fix/feature, this should not 

Reviewer: ```suggestion
The **Title** need to show a summary or a name for the new change/fix/feature, this should not 
```
